client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -242,7 +241,6 @@ class TransferDomainStep extends Component {
 		return (
 			<div>
 				<QueryProducts />
-				<QueryPlans />
```

Comment:
These need to be replaced with `QueryProducts` to ensure use currency gets populated below